docs/device_api.md

```diff
@@ -17,7 +17,7 @@ but it should be transparent for the users of the Platform.
 
 In order for the South Bound protocols (i.e.: the protocols used to communicate physical devices with the Platform)
 to work for your service, you must provision the information about your devices, either by provisioning the device itself,
-by provisioning a Configuration Group (for each subservice) or both. Configuration Groups (also called Services; do not
+by provisioning a Configuration Group (for each subservice) or both. Configuration Groups (prevously called Services; do not
 mistake them for the multitenancy Service concept), define some default values for the South Bound protocol to NGSI mapping,
```

Comment:
Consider if leaving it like this or remove the note as follows:
```suggestion
by provisioning a Configuration Group (for each subservice) or both. Configuration Groups, define some default values for the South Bound protocol to NGSI mapping,
```

Reply:
I thinks is better to remove (applying the suggestion).